build/pandoc/defaults/html.yaml

```diff
@@ -2,23 +2,30 @@
 # Load on top of common defaults.
 to: html5
 output-file: output/manuscript.html
-# include-before-body:
+include-before-body:
+  # theme
+  - build/themes/default.html
+
+  # first-party plugins
+  - build/plugins/core.html # needed for all first-party plugins
+  - build/plugins/accordion.html
+  - build/plugins/anchors.html
+  - build/plugins/attributes.html
+  # - build/plugins/inline-svg.html
+  - build/plugins/jump-to-first.html
+  - build/plugins/lightbox.html
+  - build/plugins/link-highlight.html
+  - build/plugins/table-of-contents.html
+  - build/plugins/tooltips.html
+
+  # third-party plugins
+  - build/plugins/analytics.html
 # - build/plugins/d3.html
-include-after-body:
```

Comment:
What is the reason for changing everything to be included before the body?

Reply:
For our first party plugins, it doesn't matter where they go since they all wait for the document to finish loading anyway (as if they had been placed at the end). I moved them all to one command for simplicity.

Reply:
Does anything have to go before body? I have a slight preference for after body since when people manually inspect the HTML file, it's more helpful to have the manuscript content near the top of the file.

Reply:
Afaik, D3 is the only problem child that needs to be before.

Reply:
So can we keep this as `include-before-body` contains d3 and is commented out and everything else is `include-after-body`? I see why it's a bit cleaner in the config to have a single list, but would prefer to keep the existing organization since it's been working well and I prefer after-body for manual inspection of the HTML.